src/main/resources/com/atlassian/bitbucket/jenkins/internal/deployments/DeployedToEnvironmentNotifierStep/help-environmentKey.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
is there away to use the strings from config.properties here? so that we're not doubling up